src/Controls/src/Core/Platform/Android/Extensions/TextViewExtensions.cs

```diff
@@ -62,6 +62,10 @@ internal static void SetLineBreakMode(this TextView textView, LineBreakMode line
 					break;
 				case LineBreakMode.CharacterWrap:
 					textView.Ellipsize = null;
+					if (OperatingSystem.IsAndroidVersionAtLeast(23))
+					{
+						textView.BreakStrategy = BreakStrategy.Simple;
```

Comment:
It feels risky to change this
https://developer.android.com/reference/android/text/Layout#BREAK_STRATEGY_SIMPLE

I feel like this might cause a different set of complaints from users.

I wonder if we need new APIs or some docs to cover setting this.